addons/account/populate/account_bank_statement.py

```diff
@@ -1,5 +1,7 @@
 # -*- coding: utf-8 -*-
 """Classes defining the populate factory for Bank Statements and related models."""
+import random
```

Comment:
Do not import random, but use the one provided from various functions (check how it is done elsewhere)
This is to keep the same seed in order to have a deterministic populate